app/views/application/_feedback_link.html.haml

```diff
@@ -0,0 +1,2 @@
+.feedback{tabindex:1}
```

Comment:
I know you didn't introduce the `tabindex: 1` in this PR, but it should be removed and maybe you can do that here. It forces the feedback button to _always_ come first in the tab order and fails one of axe-core's best-practice tests.